### PR TITLE
support for not automatically producing video

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**

### CODE REVIEW

File: src/useConnectCall.ts [hunk 1]
```diff
@@ -128,21 +138,23 @@ const useConnectCall = ({
...
-        const aspectRatio =
-          videoWidth && videoHeight ? videoHeight / videoWidth : undefined;
-        setLocalVideo({
-          stream,
-          paused: false,
-          aspectRatio,
-        });
-      })
-      .catch(handleError);
+    if (call?.type === CallType.VIDEO_CALL) {
```
Comment (gabrielsaruhashi) [inline src/useConnectCall.ts:141]: considered moving call.type to the constructor of RoomClient but my understanding was that RoomClient shouldn't care.

CVH will already be aware of the call type.

**Comment by gabrielsaruhashi** [comment]
@cainlevy curious to learn how this is automatically published to npm!
i see tsconfig.build etc
but what triggers the publishing?

File: src/useConnectCall.ts (hunk 1 above)
Comment (cainlevy) [reply src/useConnectCall.ts:141]: Yeah, I think we can keep this in `useConnectCall`. It actually makes me wonder if it should exist here at all. Why is this hook so opinionated about starting with video and audio? Maybe that should be the responsibility of the hook caller.

**Review by cainlevy — APPROVED** [review]
Makes sense! This raises new questions for me, but they can wait for another day (unless you feel inspired).

**Comment by cainlevy** [comment]
> what triggers the publishing?

it's manual right now. just pushed some notes to `README.md`.

**Comment by gabrielsaruhashi** [comment]
Curious to hear them out when u have some time to write it up!

**Pull request merged**